deny.toml

```diff
@@ -55,8 +55,8 @@ skip-tree = [
     # ed25519-zebra/hashbrown: https://github.com/ZcashFoundation/ed25519-zebra/pull/65
     { name = "ahash", version = "=0.7.6" },
 
-    # wait for ed25519-zebra to upgrade
-    { name = "curve25519-dalek", version = "=3.2.0" },
+    # wait for indexmap, toml_edit, serde_json, tower to upgrade
+    { name = "hashbrown", version = "=0.12.3" },
 
```

Comment:
This is interesting, I ran the command from CI which uses `--all-features` and it pinned on `0.13.2` and only because of `ed25519-zebra` and `metrics-exporter-prometheus`; is the one in CI incorrect?